opentelemetry-sdk/src/opentelemetry/sdk/util/__init__.py

```diff
@@ -41,7 +41,27 @@ def get_dict_as_key(labels):
     )
 
 
-class BoundedList(Sequence):
+def _import_config_components(
```

Comment:
As discussed offline, would probably be better suited in a `util` under `_configuration` instead of here.

Reply:
Same circular import issue. Needs to be separate from configuration.